src/App.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import UserProfile from './components/UserProfile';
+import DummyLogin from './components/DummyLogin';
+
+class App extends Component {
+  render () {
+    return (
+      <Router>
+        <Switch>
+          <Route exact path="/" component={UserProfile} />
+          <Route exact path="/login" component={DummyLogin} />
+        </Switch>
```

Comment:
So the reason for switch is to loop through the routes and when a route provided by a user is not a valid route, it falls back to the not-found route. If I may ask why is it that the not found route is not here. I feel it should be included when specifying our routes. For example, Route component={Notfound} / should be included in the list of the routes. What do you think?

Reply:
Sure. That is outside the scope of my story though.

Reply:
@TheDrizzyWay I can see that you defined the routes in the App.js components but the codebase would be neat if the routes can be in a separate file and then import it into App.js.

Reply:
True. Done

Reply:
@TheDrizzyWay I think it will be good if a fallback route is included

Reply:
@TheDrizzyWay I think it will be good if a fallback route is included

Reply:
@samorahno It's outside the scope of my story. I will ask kelechi during the TTL interaction though.